styles/main.css

```diff
@@ -20,6 +20,8 @@ a:visited {
 }
 nav ul a:link {
     color: #D49C6A;
+}
+nav ul {

```

Comment:
You need to apply the margin and padding to the `ul` directly to affect the list's margin and padding